CONTRIBUTING.md

```diff
@@ -3,6 +3,12 @@
 We welcome community contributions to this library and we hope that together we can expand the coverage of ASM ready data for everyone.
 
 In order to contribute you will need to have an Individual or Corporate Contributor License Agreement (CLA) on file with Benchling depending on if you are contributing on your own time or as part of another company. When you make your first pull request we will check if you have a CLA and if not take care of that with you first. The process is quick and painless and helps us to make sure that you and everyone who uses your code in the future is protected.
+## GPG keys and signed commits
+This repository uses signed commits. Please follow these links to get set up.
+- Check for [existing GPG keys](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits)
+- Otherwise, [generate a new GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)
+- [Add your GPG key to GitHub](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account)
+- [Tell Git about your signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key)
```

Comment:
can you add a note here that we only need to follow the steps up to step 5 for this link? 

also on the next line, could you put: "To configure commits to be signed by default within this repo, run `git config commit.gpgsign true`"